Tests/SwiftDocCTests/Diagnostics/DiagnosticTests.swift

```diff
@@ -140,6 +141,27 @@ class DiagnosticTests: XCTestCase {
         \(expectedNoteLocation): note: The message of the note.
         """)
     }
+
+    func testDoxygenDiagnostic() throws {
```

Comment:
Since this is only testing `DocumentationNode..contentFrom(documentedSymbol:documentationExtension:engine:)` the test setup could be simplified. 

You can get the symbol by decoding the symbol graph file with a `JSONDecoder()`, using the symbol's known precise identifier to find it, and then pass that symbol as input.

Reply:
Alternatively, you could create the `Symbol` directly in this test with various documentation. That would make it easier when reading this code to identify which directive is checked in each test assertion (since the doc comment isn't in a separate file anymore).

Reply:
Thank you David, I used the approach of creating Symbol, please take a look.